### DISCUSSION TITLE
What is the difference between floating menu and docked menu?

### DISCUSSION THREAD

**Discussion by mustafatoktas** [valinet/ExplorerPatcher#3300] [Q&A]
🟢Rounded corners, floating menu
🟢Rounded corners, docked menu
What is the difference between these two settings?

![331831774-605513fa-c36c-4705-a055-03f127220ff4](https://github.com/valinet/ExplorerPatcher/assets/52534609/09341822-cce3-480f-b64c-55831a71d8a6)

🟢I know the difference between not rounded and rounded corners. I can also see.
🟢I don't understand the difference between `floating menu` and  `docked menu`. Is there any chance you can explain it with a visual or a short video?

| **Not rounded** | **Rounded corners** |
|     --------------------       |     ------------------     |
|  ![1](https://github.com/valinet/ExplorerPatcher/assets/52534609/117c3bc1-87e8-4714-a89c-ff71c7660827) |  ![2](https://github.com/valinet/ExplorerPatcher/assets/52534609/dbbf7f5e-0014-4c3a-bc87-c9ecdb4e71cb) |

**Comment by pyrates999** [comment]
you can try enabling floating menu to see what the difference is.

**Reply by mustafatoktas** [reply]
@pyrates999 
## Do you think I haven't tried? You're really funny, my friend.
It's interesting that you actually think I haven't tried; I'm concerned about your mental health. Why would I bother writing here if I hadn't tried?

I tried and couldn't see the difference. If you're going to explain what the difference is, then go ahead, otherwise don't bother engaging with me.

**Reply by pyrates999** [reply]
your response is not helpful.
